ClinGen/clincoded
# Connects to #89. Experimental Data curation page

* Added experimental data curation page (excluding Assessments; see #337)
* Added relevant experimental data information to Dashboard and Curation Central
* Expanded forms.js to support checkboxes
* Updated schema (spelling corrections and text changes)

Testing:
1. Create GDM
2. Add PMID
3. Add Experimental Data one of each Experimental Data type (two for Biochemical Function and Expression: one for A subtype, one for B subtype)
4. Confirm that the Variants panel DOES NOT exist for Biochemical Function, Protein Interaction, and Expression subtype A
5. Confirm that the entries show up correctly in Curation Central (including associated Variants)
6. Confirm that the entries show up correctly on Dashboard
7. Check Edit pages for these Experimental Data entries
8. Check View pages for these Experimental Data entries
